engine/pkg/meta/util.go

```diff
@@ -27,7 +27,7 @@ func CreateSchemaIfNotExists(ctx context.Context, storeConf model.StoreConfig) e
 	schema := storeConf.Schema
 	// clear the schema to open a connection without any schema
 	storeConf.Schema = ""
-	db, err := dbutil.NewSQLDB("mysql", model.GenerateDSNByParams(&storeConf, nil), nil /*DBConfig*/)
+	db, err := sqlkv.NewSQLDB(&storeConf)
```

Comment:
I think Implementing  the 'tls' in 'dbutil' is more proper.

Reply:
I met cycle import 😂

Reply:
Using some specific `sqlkv` implement in a more common `util.go` sounds wierd.

Reply:
`CreateSchemaIfNotExists` should only be called for sqlkv. You can check your PR #6435

Reply:
> I met cycle import 😂 We can do it in another PR

I revert the change in this file and find it compile well.

Reply:
> `CreateSchemaIfNotExists` should only be called for sqlkv. You can check your PR #6435

Yes. `CreateSchemaIfNotExists` should only be called for sqlkv, but binding it with the implement in `internal` still wierd.

Reply:
> > I met cycle import 😂 We can do it in another PR
> 
> I revert the change in this file and find it compile well. ??

You didn't enable TLS functionality, so of course it compile well since it's same as original code 🤷